### PR TITLE
BUILD: Include debian files in source tarball.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2895/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/377/ for details.

**Comment by miked-mellanox** [comment]
@yosefe - spec license - do you plan t fix it in separate PR or can be same one.

**Comment by yosefe** [comment]
will update this pr. there are also some failures during build on debian.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2308/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/378/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2896/ for details.

File: ucx.spec.in [hunk 1]
```diff
@@ -19,7 +19,7 @@ Summary: Unified Communication X
 Version: %{version}
 Release: %{rel}
 
-License: Proprietary
+License: BSDv3
```
Comment (shamisp) [inline ucx.spec.in]: Since you change the file, please change the name from openucx.org -> www.openucx.org

File: debian/control.in [hunk 2]
```diff
@@ -1,15 +1,15 @@
 Source: @PACKAGE@
 Section: libs
 Priority: extra
-Maintainer: support@openucx.org
+Maintainer: ucx-group@elist.ornl.gov
 Build-Depends:
 Standards-Version: @MAJOR_VERSION@.@MINOR_VERSION@
 Homepage: http://openucx.org
```
Comment (shamisp) [inline debian/control.in]: www.openucx.org

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2309/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2898/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2311/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/383/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/384/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/385/ for details.

**Comment by yosefe** [comment]
@shamisp your comments is addressed
tests are failing because of temporary setup issues and unrelated problem
since this PR does not change actual source files, merging this

**Pull request merged**